app/src/main/java/org/oppia/android/app/help/RouteToPrivacyPolicyListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.app.help
+
+/** Listener for when a selection should result to [PrivacyPolicyActivity]. */
+interface RouteToPrivacyPolicyListener {
+  /**  Called when the user wants to open the Privacy policy page. */
```

Comment:
```suggestion
  /**  Called when the user wants to open the privacy policy page. */
```

Ditto in RouteToTermsOfServiceListener (for capitalization).

Reply:
Done.

Reply:
Done.